test/functional/kernel/nested-loop-reducesum/test-kernel-nested-loop.cpp.in

```diff
@@ -72,6 +78,9 @@ using CudaKernelNestedLoopExecPols = camp::list<
   
 using HipKernelNestedLoopExecPols = camp::list<
 
+    // Device Depth 1 ReduceSum Exec Pols
+    NestedLoopData<DEVICE_DEPTH_1_REDUCESUM, RAJA::hip_thread_x_loop, RAJA::hip_block_reduce >,
+
```

Comment:
No openmp?

Reply:
There is no kernel::openmp reduction available. Same goes for OmpTarget, and TBB.